teachertool/src/components/CriteriaInstanceDisplay.tsx

```diff
@@ -158,7 +157,10 @@ export const CriteriaInstanceDisplay: React.FC<CriteriaInstanceDisplayProps> = (
                     </span>
                 ))}
             </div>
-            <div className={css["criteria-description"]}>{catalogCriteria.description}</div>
+            <div className={classList(css["criteria-description"], "no-print")}>{catalogCriteria.description}</div>
+            <div className={classList(css["criteria-description"], css["for-print"], "only-print")}>
```

Comment:
I don't know much about making webpages printable. Is this a common pattern to an element defined twice, one with classes for printing and one without?

Reply:
Just seemed like a clean way to implement, given how we've setup the media queries. Open to other suggestions!

Reply:
If italics is the only difference, putting it on the same control and just using a print media query in the CriteriaInstanceDisplay.module.scss would feel slightly cleaner to me, but I don't feel strongly.

Reply:
I generally shy away from sprinkling media queries around css, as they introduce hidden behaviors not apparent in code.

Reply:
I agree with Thomas that a media query feels cleaner

Reply:
Are the "only-print" and "no-print" css classes not clear?

Reply:
I'd say it's clear enough. Just a personal stylistic preference. If you prefer this approach, that's fine by me!

Reply:
Ah, I see your point around media queries. 
The "only-print" and "no-print" classes are definitely clear, but it just feels a bit funky to have two divs defined here where the inner content is the same. That said, I think your stance about media queries makes sense, so I'm fine with leaving it as is.

Reply:
I see your points. I think I'd rather leverage the system we have rather than make an exception here.

Reply:
Sounds good!